templates/alerts.html

```diff
@@ -22,13 +22,15 @@
             var row = tbl.insertRow(-1);
             row.id = alert["id"] + "-";
             var rev = row.insertCell(0);
+            rev.className="check-alert";
+            rev.innerHTML="<input type=\"checkbox\" id =\"check-"+alert['id']+"\"/>";
 
             for (var i = 0; i < fields.length; i++) {
                 var test = fields[i];
                 cell = row.insertCell(-1);
                 cell.className = cname;
                 cell.setAttribute('field_name', fields[i]);
-                cell.onclick = editAlert(alert["id"], alert["body"]);
+                // cell.onclick = editAlert(alert["id"], alert["body"]);
```

Comment:
is there a reason for this?  I assume this gets rid of the popup whenever you click on the alert row.  I would prefer to remove the line entirely then!